### PR TITLE
Bug 1909096: Fix missing key errors for noobaa and ceph

### PR DESCRIPTION
**Pull request by afreen23**
- on the review page of internal mode
- on select capcaity and nodes page of internal mode
- on backing store step of bucket class

Signed-off-by: Afreen Rahman <afrahman@redhat.com>

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx [hunk 1]
```diff
@@ -26,8 +26,7 @@ const BackingStorePage: React.FC<BackingStorePageProps> = React.memo(
           >
             <p>
               {t(
-                'noobaa-storage-plugin~Backing Store represents a storage target to be used as the underlying storage for the data in {{MCG}} buckets.',
-                { mcg: 'MCG' },
+                'noobaa-storage-plugin~Backing Store represents a storage target to be used as the underlying storage for the data in MCG buckets.',
```
Comment (bipuladh) [inline frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx:29]:
```suggestion
                'noobaa-storage-plugin~Backing Store represents a storage target to be used as the underlying storage for the data in MultiCloud Object Gateway buckets.',
```
Please fix all instances of this. Thanks

**Review by bipuladh — CHANGES_REQUESTED** [review]

File: frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx (hunk 1 above)
Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx:29]: This can be handled in a separate PR.

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**